ui/main/src/shared/css/opfab-application.css

```diff
@@ -614,4 +615,11 @@ input:-webkit-autofill {
 .opfab-btn-cancel:hover {
     color: #2268ff;
 }
+
+.opfab-bubble-text{
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    max-width: 600px;
+}
```

Comment:
to be add in ui/main/src/scss/styles.scss instead (this style is not shared with external applications)